http-add-on/templates/deployment-operator.yaml

```diff
@@ -39,6 +39,8 @@ spec:
         - --logtostderr=true
         - --v=10
         image: "{{ .Values.images.kubeRbacProxy.name }}:{{ .Values.images.kubeRbacProxy.tag }}"
+        resources:
+          {{- toYaml .Values.resources | nindent 12 }}
```

Comment:
Nice catch but I think that we can hard-code the values because it's only for the metrics proxy, otherwise we will request too much resources for it

Reply:
good point. Just changed it. Does this look better?

Reply:
yes it does, maybe we need to increase or allow updates in the future, but as starting point is good enough